lib/Installation/Popups/UntrustedGPGPopup.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces methods to handle
+# a Trust&Import popup.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Popups::UntrustedGPGPopup;
```

Comment:
we always try to use the name of the screen:
s/UntrustedGPGPopup/ImportUntrustedGnuPGKey/ (not much longer)